readthedocs/projects/tasks/utils.py

```diff
@@ -154,6 +160,81 @@ def send_external_build_status(version_type, build_pk, commit, status):
         send_build_status.delay(build_pk, commit, status)
 
 
+class DeprecatedConfigFileSiteNotification(SiteNotification):
+
+    failure_message = (
+        "Your project '{{ object.slug }}' doesn't have a "
+        '<a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">.readthedocs.yaml</a> '
+        "configuration file. "
+        "This feature is <strong>deprecated and will be removed soon</strong>. "
+        "Make sure to create one for your project to keep your builds working."
```

Comment:
This one just sounded a little terse or something to me, but I'm not sold on my own edit either.

```suggestion
        "Make sure to create one for your project to ensure your project continues building successfully."
```